packages/EmojiPicker/src/styles.ts

```diff
@@ -3,6 +3,7 @@ import { Tab } from '@welcome-ui/tabs'
 import * as WUIPopover from '@welcome-ui/popover'
 import { Box } from '@welcome-ui/box'
 
+// this component is imported by WH but not used, could be deprecated
```

Comment:
@P1X3L As Robert showed me, this component is not used. Maybe we can deprecate it?

I have the remaining TS error on it

Reply:
Yes we can deprecated for now just ignore ts on this file :) I'll create a ticket for the deprecatation

Reply:
Done 🙂